third_ex.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+
+def create_and_print_number_of_rows_and_cols() -> None:
+    """
+    This function creates an array and prints the amount of rows and
+    columns that this array has.
+    """
+    my_array = np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
+    number_of_rows, number_of_columns = my_array.shape
+
+    print("Rows = ", number_of_rows)
+    print("Columns = ", number_of_columns)
```

Comment:
File should end with a newline - 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline